TrackerConditions/test/MisalignTracker.C

```diff
@@ -26,9 +26,10 @@ double PanelPhi(int plane, int panel){
 }
 
 
```

Comment:
Do we really need these bools?  Ive tested that, if provide with 0.0 values, the output .txt files gives 0.0 for the misalignments.

Reply:
It was convenient for me when testing - maybe that was back when a lot of the parameters were shared, it may not be needed anymore